README.md

```diff
@@ -1 +1,41 @@
-# to-do-list
\ No newline at end of file
+# to-do-list
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> Web page to Showcase an Interactive To-do List as Microverse Project.
+
+![screenshot](src/Snapshot.png)
+
+[Online Version]()
+
```

Comment:
You're missing information in your README file. You could use [this template](https://github.com/microverseinc/readme-template).

- Prerequisites
- Setup
- Install
- Usage

Kindly work on these changes and request another code review in your dashboard.

Reply:
Empty headings?? Really????  🤦

Reply:
- Kindly in your General requirements of your To Do list: list structure you will find this section:
Make sure that you documented your work [in a professional way.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
- Kindly check this point:
 5.- Add descriptive README file to your project - please use this[ template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
 
Any question that you have or if you fell stuck, I can support you on slack.
Greetings!.
@carlos-ssh 👾🥷🏽👨🏽‍💻